plutus-benchmark/marlowe/src/PlutusBenchmark/Marlowe/Scripts/Semantics.hs

```diff
@@ -317,7 +317,7 @@ mkMarloweValidator
         $ checkScriptOutput (==) ownAddress hsh ocValue getContinuingOutput
 
     getContinuingOutput :: TxOut
-    getContinuingOutput = case filter (\TxOut{txOutAddress} -> ownAddress == txOutAddress) allOutputs of
+    ~getContinuingOutput = case filter (\TxOut{txOutAddress} -> ownAddress == txOutAddress) allOutputs of
```

Comment:
I'm not sure I agree - it's just what you'd expect in a strict language! You have a binding of a value (not a function), so that's going to get evaluated strictly. It's only surprising if you're used to Haskell, where you can put let-bindings wherever you want without consequences.